src/unit_tests/wrappers/wazuh/wazuh_db/wdb_global_wrappers.c

```diff
@@ -268,3 +268,23 @@ int __wrap_wdb_global_agent_exists(wdb_t *wdb, int agent_id) {
     check_expected(agent_id);
     return mock();
 }
+
+cJSON* __wrap_wdb_global_get_backups() {
+    return mock_ptr_type(cJSON*);
+}
+
+int __wrap_wdb_global_create_backup(__attribute__((unused)) wdb_t* wdb,
+                                    char* output,
+                                    const char* tag) {
+    snprintf(output, OS_MAXSTR + 1, "%s%s", mock_ptr_type(char*), tag ? tag : "");
```

Comment:
Maybe we are not using it right now. But we should leave the possibility to be used in the future in case of having the need of adding new test cases.